build_frontends.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -ex
+
+echo "Install latest dependencies and build both maker and taker frontends"
+
+cd maker-frontend
+yarn install
+yarn build
+cd ..
+cd taker-frontend
+yarn install
+yarn build
+cd ..
+echo "Maker and taker frontends built successfully"
```

Comment:
nit: This line may lie... 😄 If you really want to print a success message you should look at the result of the commands above 😉

Reply:
sorry, how can this lie? I'm using `set -e`, which will stop this script on error

Reply:
Ups, I've overlooked that..  🙈